services/media/service/src/ingest/handlers/image-succeeded-handler.ts

```diff
@@ -106,19 +86,16 @@ export abstract class ImageSucceededHandler<
     }
     const messageContext = metadata.messageContext as ImageMessageContext;
 
-    await this.storeInboxMessage<CheckFinishIngestItemCommand>(
-      messageContext.ingestItemId.toString(),
-      MediaServiceMessagingSettings.CheckFinishIngestItem,
+    await update(
+      'ingest_item_steps',
       {
-        ingest_item_step_id: messageContext.ingestItemStepId,
-        ingest_item_id: messageContext.ingestItemId,
-        error_message: getIngestErrorMessage(
+        status: 'ERROR',
+        response_message: getIngestErrorMessage(
           error,
           'An unexpected error occurred while trying to update image relations.',
```

Comment:
side note: could we maybe update this to something like 

* The image [was correctly imported | already existed] but there was an error adding that image to the entity.

Reply:
Will update in context of other upcoming error-handling-related tasks.